### DISCUSSION TITLE
STAR issues..

### DISCUSSION THREAD

**Discussion by Annolago** [flybywiresim/aircraft#4890] [Q&A]
Hi
So I have an issue with how the A320 is flying a STAR on AP. As an example, take EDDH, the BOGMU 1P RNAV arrival. 
It is basically a series of turns going right, left, left, right, left and then right again.
There is a set of restrictions in terms of altitude and the snake pattern is started with a possibility for a hold pattern and then once entered the idea is to drop altitude from below FL110 to above 4000. The same pattern exist many places - Rome Fiorino as another example. 
Now the issue is that if I code this in the FMU then the place just cheats and fly in almost a straight line to the end point. 
Now have seen this many other places where a pattern designed to drop speed and altitude is just "ignored" and straightened out.
Now this may be not yet implemented etc - but would love to know if this is working as designed or if something can be done about it, except from flying it manually by selecting heading, as I am forced to doing now.
Thanks,,,

**Comment by ExampleWasTaken** [comment]
The A32NX does not currently adhere to altitude or speed restrictions. It should follow your flight plan laterally. It's a known issue that it may turn direct to the airport on approach.

These features are being worked on at the moment.
